src/templates/finops-hub/modules/keyVault.bicep

```
@@ -14,8 +14,9 @@ param location string = resourceGroup().location
 @description('Optional. Array of access policies object.')
 param accessPolicies array = []
 
-@description('Required. Name of the storage account to store access keys for.')
-param storageAccountName string
+@description('Optional. Name of the storage account to store access keys for.')
```

Is this outdated?
```suggestion
@description('Optional. Storage account key to store as a secret.')
```
(Feel free to use a better description)

---

In that case, we should probably name it more clearly. I was about to ask that. Guess I should've followed thru with my initial instincts 😜